config/local_env.yml

```diff
@@ -0,0 +1,14 @@
+# Weights of individual activities
+WEIGHT_SOLVED: '5'
+WEIGHT_FAILED: '2'
+WEIGHT_DONT_NOW: '1'
+WEIGHT_COMMENT: '3'
+
+# Base for the exponential function that which reduces the contribution of  actual comment to the score for the room
+BASE_EXP_FUNCTION: '0.9'
+
+# The minimal percentage success required in the room
+MIN_PERCENTAGE: '0.6'
+
+# The number of LOS in room
+NUMBER_LOS: '10'
```

Comment:
Kedze su to globalne premenne, tieto nazvy nie su najstastnejsie. `NUMBER_LOS` v com? Chcelo by to nejake prefixy aspon, alebo ist konkretnejsie (napr. toto je pocet LO v miestnosti, nie?).